src/scenekit.cs

```diff
@@ -1784,15 +1790,13 @@ interface SCNSceneLoadingOptions {
 		[Export ("SCNSceneSourceLoading.OptionPreserveOriginalTopology")]
 		bool PreserveOriginalTopology { get; set; }
 
-#if !TVOS && !WATCH
 		// note: generator's StrongDictionary does not support No* attributes yet
```

Comment:
This comment seems important: does the generator's StrongDictionary support No* attributes now? If not, we can't remove the #if conditions (and if it does, then you can remove the comment).

Reply:
@chamons  would you happen to know this one?

Reply:
I would go check the generated code and see what it generates before and after your change.